go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/prometheus/client_golang v1.8.0
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
+	github.com/submariner-io/admiral v0.7.0
```

Comment:
We should just import the latest admiral. Also, we'll need to update Step 4 in the [Release Process](https://submariner.io/contributing/release-process/) doc to include the admiral dependency.

Reply:
We're close enough to Rc0, so it can come when we bump up.

Reply:
Done